test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -628,34 +639,61 @@ pragma experimental ABIEncoderV2;)";
 		.render();
 }
 
+string ProtoConverter::isabelleTypeString() const
+{
+	string typeString = m_isabelleTypeString.str();
+	if (!typeString.empty())
+		return "(" + typeString + ")";
+	else
+		return typeString;
+}
+
 string ProtoConverter::contractToString(Contract const& _input)
 {
 	visit(_input);
 	return m_output.str();
 }
 
 /// Type visitor
+void TypeVisitor::StructTupleString::addTypeStringToTuple(string& _typeString)
+{
+	index++;
+	stream << Whiskers(R"(<?isNotFirstElement>,</isNotFirstElement><typeString>)")
```

Comment:
This one is also arguable ;-). But fine to merge it like that, but you can think about changing it now or in the follow-up PR as well.

Reply:
Thanks, updated!